blueocean-core-js/src/js/fetch.js

```diff
@@ -290,7 +290,10 @@ function getPrefetchedDataFuture(url) {
                 // the prefetched data as the value.
                 if (trimRestUrl(preFetchEntry.restUrl) === trimmedUrl) {
                     try {
-                        return Promise.resolve(JSON.parse(preFetchEntry.data));
+                        const parse = JSON.parse(preFetchEntry.data);
+                        return Promise.resolve(parse);
+                    } catch (e) {
+                        console.error('preFetch Error', e);
```

Comment:
Perhaps we should actually rethrow the error with prefetch.data attached, so we can see it in rollbar? My understanding is that we haven't been able to repro this locally, but if it shows up in rollbar then probably easier for us to pin down what's going on.